internal/indexcoord/index_coord.go

```diff
@@ -538,6 +538,68 @@ func (i *IndexCoord) GetSegmentIndexState(ctx context.Context, req *indexpb.GetS
 	return ret, nil
 }
 
+// CompleteIndexInfo get the building index progress and index state
+func (i *IndexCoord) completeIndexInfo(ctx context.Context, indexInfo *indexpb.IndexInfo) error {
+	collectionID := indexInfo.CollectionID
+	indexName := indexInfo.IndexName
+	log.Info("IndexCoord completeIndexInfo", zap.Int64("collID", collectionID),
+		zap.String("indexName", indexName))
+	flushSegments, err := i.dataCoordClient.GetFlushedSegments(ctx, &datapb.GetFlushedSegmentsRequest{
+		CollectionID: collectionID,
+		PartitionID:  -1,
+	})
+	if err != nil {
+		return err
+	}
+
+	resp, err := i.dataCoordClient.GetSegmentInfo(ctx, &datapb.GetSegmentInfoRequest{
+		SegmentIDs: flushSegments.Segments,
+	})
+	if err != nil {
+		return err
+	}
+	totalRows, indexRows := int64(0), int64(0)
+
+	for _, seg := range resp.Infos {
+		totalRows += seg.NumOfRows
+	}
+
+	indexID2CreateTs := i.metaTable.GetIndexIDByName(collectionID, indexName)
+	if len(indexID2CreateTs) < 1 {
```

Comment:
Please add ut for this case. Especially these error cases should be payed more attention.

Reply:
These codes are previous, and I just move it.